apps/backend/src/api/routes/v1/auth_profile.py

```
@@ -0,0 +1,138 @@
+from src.middleware.auth import get_current_user, require_auth
+from src.models.user import User
+
+logger = logging.getLogger(__name__)
+
+router = APIRouter()
+user_service = UserService()
+
+
+@router.get(
+    "/me",
+    response_model=Union[UserResponse, ErrorResponse],
```

Ruff (0.11.9)

Use `X | Y` for type annotations

(UP007)